Honeywell dimmer and switch fingerprints

greens opened the pull request:
WWST-159 WWST-155 WWST-164 WWST-157 WWST-166 WWST-158 WWST-156 WWST-165 WWST-160 WWST-167

Add fingerprints for Honeywell dimmers and switches.

mckeed committed:
Add mfr fingerprints to Fibaro Motion
NVS-6

workingmonk committed:
Merge pull request #2596 from SmartThingsCommunity/master

Rolling up master to staging

varzac committed:
Mark 0.20.X devices to run locally

This is marking the hubcore 0.20.X newly supported devices to run
locally.

This resolves: https://smartthings.atlassian.net/browse/CHAD-1633

workingmonk committed:
Merge pull request #2602 from varzac/hubcore-20-local-devices

[CHAD-1633] Mark 0.20.X devices to run locally

dkirker committed:
DVCSMP-3349 Use old reporting logic for older firmware Centralite sensors

greens committed:
WWST-159 WWST-155 WWST-164 WWST-157 WWST-166 WWST-158 WWST-156 WWST-165 WWST-160 WWST-167

Add fingerprints for Honeywell dimmers and switches.

greens committed:
WWST-159 WWST-155 WWST-164 WWST-157 WWST-166 WWST-158 WWST-156 WWST-165 WWST-160 WWST-167

Add fingerprints for Honeywell dimmers and switches.

workingmonk committed:
Merge pull request #2603 from dkirker/DVCSMP-3349

DVCSMP-3349 Use old reporting logic for older firmware Centralite sensors

munds committed:
Update zigbee fingerprints for sengled

Add Sengled E12-N15 to zigbee dimmer DTH

Reorder zigbee fingerprinting lines alphanumerically.

workingmonk committed:
Merge pull request #2615 from munds/DVCSMP-3471

[DVCSMP-3471] Update zigbee fingerprints for Sengled

juano2310 committed:
NQA-793 - Thermostat automation support (#2614)

Small typos

small change

greens committed:
WWST-170 Honeywell device fingerprints

greens committed:
WWST-170 Honeywell device fingerprints

greens commented:
Added the last fingerprint for the fan control.

greens committed:
DVCSMP-1524 Remove polling capability from hub-polled devices (#2510)

Removes the deprecated polling capability from these devices which are polled by the hub directly.

dkirker reviewed (dismissed):
Looks good! Only thing I would recommend is the squash the commits to one. The Vinay will request this.

workingmonk committed:
Merge pull request #2620 from SmartThingsCommunity/staging

Rolling down staging to master

mckeed committed:
Merge pull request #1765 from mckeed/fibaro-motion-mfr

Add mfr fingerprints to Fibaro Motion

greens committed:
Dvcsmp-1437 GE Fan switches don't support configuration (#2626)

* DVCSMP-1437 GE Fan switches don't support indicator LED control

Move the GE Fan switch to the generic dimmer handler, since they don't support the configuration options of the other GE dimmers.

greens committed:
WWST-162 Dimmer switches also not polling for initial status

This adds an initial poll on install like we have for (non-dimmer) zwave switches.

greens committed:
Merge branch 'WWST-155' of https://github.com/greens/SmartThingsPublic into WWST-155

greens commented:
tried to rebase and shot myself in the foot
